Heart/User/Presentation/UsersController.php

```diff
@@ -26,4 +27,9 @@ public function getUser(int $id, GetUser $getUser): JsonResponse
             );
         }
     }
+
+    public function getProfile(string $value, FindProfile $profile): JsonResponse
+    {
+        return response()->json($profile->handle($value));
```

Comment:
Colocar um status code no ```return response()->json($profile->handle($value));```, não sei se é necessario, pode ser um OK, ou um noContent. 
```suggestion
        return response()->json($profile->handle($value), Response::HTTP_NO_CONTENT);
```